edb/schema/pointers.py

```diff
@@ -896,6 +896,8 @@ def _parse_computable(
 
         source = schema.get(source_name, type=s_objtypes.ObjectType)
 
+        ptr_name = sn.shortname_from_fullname(
```

Comment:
Please don't open-code object names, use standard helpers: `get_displayname_static()`, `get_verbosename_static()` etc. For this case you can add a shortcut similar to `get_verbosename()` on `ObjectCommand` to get `displayname`.